mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -77,189 +79,112 @@ func TestClientSideEncryptionProse(t *testing.T) {
 	}
 
 	runOpts := mtest.NewOptions().MinServerVersion("6.0").Topologies(mtest.ReplicaSet, mtest.LoadBalanced, mtest.ShardedReplicaSet)
-	mt.RunOpts("explicit encryption", runOpts, func(mt *mtest.T) {
-		// Test Setup ... begin
-		encryptedFields := readJSONFile(mt, "encrypted-fields.json")
-		key1Document := readJSONFile(mt, "key1-document.json")
-		var key1ID primitive.Binary
+	mt.Run("1. custom key material test", func(mt *mtest.T) {
+		const (
+			dkCollection = "datakeys"
+			idKey        = "_id"
+			kvDatabase   = "keyvault"
+		)
+
+		var cse *cseProseTest
+
+		// Create a ClientEncryption object (referred to as client_encryption) with client set as the keyVaultClient.
+		// Using client, drop the collection keyvault.datakeys.
 		{
```

Comment:
I don't think the `{` indents are particularly helpful for readability, but I do not feel too strongly. Keep them if you wish and @kevinAlbs is fine with them.